pkg/kubelet/volumemanager/volume_manager.go

```diff
@@ -479,6 +487,25 @@ func (vm *volumeManager) WaitForUnmount(ctx context.Context, pod *v1.Pod) error
 	return nil
 }
 
+func (vm *volumeManager) WaitForAllPodsUnmount(ctx context.Context, pods []*v1.Pod) error {
+	var (
+		errors []error
+		wg     sync.WaitGroup
+	)
+	wg.Add(len(pods))
+	for _, pod := range pods {
+		go func(pod *v1.Pod) {
+			defer wg.Done()
+			if err := vm.WaitForUnmount(ctx, pod); err != nil {
+				klog.V(5).ErrorS(err, "Failed to wait for Pod to unmount its volumes", "pod", klog.KObj(pod))
```

Comment:
Why `V(5).Error`? Are you thinking it will be so often that we want to hide this error in a default logging level?

Reply:
Removed this log statement in the newest rev - the error gets returned to the caller and logged there at the default logging level.